app/models/conference.rb

```diff
@@ -95,9 +95,11 @@ def confirmed
 
   validates_uniqueness_of :short_title
   validates_format_of :short_title, with: /\A[a-zA-Z0-9_-]*\z/
+  validates :registration_limit, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   # This validation is needed since a conference with a start date greater than the end date is not possible
   validate :valid_date_range?
+  validate :registration_limit_not_exceed
```

Comment:
I have added the tests. I need to change the comparision in registrations_limit_exceeded? to >= instead of == so the validation to work.